### ISSUE TITLE
OCTune resets core delta to 0 when changing one of the other values

### ISSUE THREAD

**Issue by gigaplex** [nicehash/NiceHashQuickMiner#235]
0.4.1.3 stable build of QuickMiner
NVIDIA drivers 461.72

I have my OCTune settings set to -500 core, +1100 mem, 120W power limit. When I try to fine tune either mem or power, ie bumping mem up to 1150 and pressing "Single" to apply it, the logs say it's setting the card to -500, +1150, 120W but the card actually starts running at 0, +1150, 120W. If I then change the core value to something else, eg -450, it'll then successfully run at -450, +1150, 120W. If I then change both the core and mem simultaneously to -500, +1100, 120W, it'll successfully change and run properly at those settings.

TL;DR I have to change the core offset value to a different number than the current setting while changing either of the other values, otherwise it'll actually run the core at 0 offset.

**Comment by gigaplex** [comment]
![image](https://user-images.githubusercontent.com/32552/111618168-27127800-8838-11eb-83f9-12757ff05eb6.png)

**Comment by nicehashdev** [comment]
I have noticed similar issues myself, will def. look into this and will be fixed for next version.

**Comment by moops44** [comment]
Make a copy from commands.json and nhqm.conf somewhere on your pc
Delete the orginal commands.json, start quickminer , open octune , do random settings and then save the current settings,
check the quickminer folder if a new commands.json file is created , if not uninstall quickminer with Revo Uninstaller Pro to delete every file which belongs to quickminer.
Reinstall Quickminer , **_dont open it_** , Paste the two files you copied before in the Quickminer Folder , and try your fine settings again

**Comment by nicehashdev** [comment]
Is this issue still present in latest RC version?

**Comment by gigaplex** [comment]
Seems to work fine on the latest RC version.

Side note: upgrading to the latest RC version could be a little easier. Trying to run the installer again to tell it to install the RC just throws an error that it's already installed. It would be nice if it could allow reinstalling instead of throwing an error.

**Comment by nicehashdev** [comment]
Which installer did you use? This bug was fixed long time ago. Just make sure to use recent installer and not the old one.
